pkg/enpass/vault.go

```diff
@@ -177,8 +177,8 @@ func (v *Vault) GetEntries(cardType string, filters []string) ([]Card, error) {
 			continue
 		}
 
-		// if we specify a type filter
-		if cardType != "" && card.Type != cardType {
```

Comment:
Perhaps it would still be nice to return everything if no type is set, since we have no way to derive all available types for the user.